lib/gridfs-stream/upload.js

```diff
@@ -447,3 +486,15 @@ function writeRemnant(_this, callback) {
     checkDone(_this);
   });
 }
+
+/**
+ * @ignore
+ */
+
+function checkAborted(_this, callback) {
+  if (_this.state.aborted) {
+    callback(new Error('this stream has been aborted'));
```

Comment:
`callback` may be undefined, at least via its `end()` caller, possibly others too.